From: Dmitry Shachnev <mitya57@debian.org>
Date: Fri, 7 Feb 2020 13:59:30 +0300
Subject: Move sphinxcontrib modules to extras_require

---
 pyproject.toml | 14 ++++++++------
 1 file changed, 8 insertions(+), 6 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index 0a9bc9e..6383e31 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -55,12 +55,6 @@ classifiers = [
     "Topic :: Utilities",
 ]
 dependencies = [
-    "sphinxcontrib-applehelp",
-    "sphinxcontrib-devhelp",
-    "sphinxcontrib-jsmath",
-    "sphinxcontrib-htmlhelp>=2.0.0",
-    "sphinxcontrib-serializinghtml>=1.1.9",
-    "sphinxcontrib-qthelp",
     "Jinja2>=3.0",
     "Pygments>=2.14",
     "docutils>=0.18.1,<0.21",
@@ -96,6 +90,14 @@ test = [
     "setuptools>=67.0",  # for Cython compilation
     "filelock",
 ]
+sphinxcontrib = [
+    "sphinxcontrib-applehelp",
+    "sphinxcontrib-devhelp",
+    "sphinxcontrib-jsmath",
+    "sphinxcontrib-htmlhelp>=2.0.0",
+    "sphinxcontrib-serializinghtml>=1.1.9",
+    "sphinxcontrib-qthelp",
+]
 
 [[project.authors]]
 name = "Georg Brandl"
