From: Dmitry Shachnev <mitya57@debian.org>
Date: Tue, 20 Dec 2016 16:37:32 +0300
Subject: Skip tests that require network access

Forwarded: not-needed
---
 tests/test_build_latex.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tests/test_build_latex.py b/tests/test_build_latex.py
index f6c3369..5996203 100644
--- a/tests/test_build_latex.py
+++ b/tests/test_build_latex.py
@@ -1420,6 +1420,7 @@ def test_latex_raw_directive(app, status, warning):
     assert 'LaTeX: abc def ghi' in result
 
 
+@pytest.mark.skip('Requires internet access')
 @pytest.mark.sphinx('latex', testroot='images')
 def test_latex_images(app, status, warning):
     app.builder.build_all()
