From: Dmitry Shachnev <mitya57@debian.org>
Date: Sat, 21 Dec 2024 20:43:03 +0300
Subject: Do not perform any timezone conversions if SOURCE_DATE_EPOCH is set

Forwarded: not-yet
---
 sphinx/util/i18n.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sphinx/util/i18n.py b/sphinx/util/i18n.py
index 5fca3c8..b775246 100644
--- a/sphinx/util/i18n.py
+++ b/sphinx/util/i18n.py
@@ -255,6 +255,7 @@ def format_date(
         source_date_epoch = os.getenv('SOURCE_DATE_EPOCH')
         if source_date_epoch is not None:
             date = datetime.fromtimestamp(float(source_date_epoch), tz=UTC)
+            local_time = False
         else:
             date = datetime.now(tz=UTC)
 
