From: Dmitry Shachnev <mitya57@debian.org>
Date: Tue, 4 Feb 2020 00:12:18 +0300
Subject: Use packaged MathJax (for our own documentation)

---
 doc/conf.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/doc/conf.py b/doc/conf.py
index d4915aa..416158f 100644
--- a/doc/conf.py
+++ b/doc/conf.py
@@ -185,6 +185,7 @@ nitpick_ignore = {
     ('std:confval', 'globaltoc_maxdepth'),
 }
 
+mathjax_path = 'file:///usr/share/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML'
 
 # -- Extension interface -------------------------------------------------------
 
